### DISCUSSION TITLE
tailwind css installed but not working in my webpack + react + umi project

### DISCUSSION THREAD

**Discussion by pangchao21** [tailwindlabs/tailwindcss#13645] [Help]
What version of Tailwind CSS are you using?
3.4.3
What build tool (or framework if it abstracts the build tool) are you using?
webpack
What version of Node.js are you using?
v14.16.1
What browser are you using?
chrome
What operating system are you using?
win11
Describe your issue
my project use webpack + react (i create project select vanilla js and after add react to it )
i install the tailwind css follow the guide https://tailwindcss.com/docs/guides/vite
no any step is missing , but the tailwindcss seems not work,

![1715071385039](https://github.com/tailwindlabs/tailwindcss/assets/25843968/94d4247d-384d-46f4-8fb5-4526870b969e)
![1715071425971](https://github.com/tailwindlabs/tailwindcss/assets/25843968/705ef05c-c7b1-4fa7-9184-bf84931d7b46)
![1715071494513](https://github.com/tailwindlabs/tailwindcss/assets/25843968/e5212346-6c64-4c3d-9a5d-818ea8c129fb)

**Comment by wongjn** [comment]
Consider checking the `content` file globs sufficiently cover the file that the code appears in. You could also try only `require`ing the PostCSS plugins, without executing them.

Otherwise, consider providing a project that reproduces the unexpected behavior.
